client/lib/abtest/active-tests.js

```diff
@@ -69,10 +69,10 @@ export default {
 	},
 	conciergeUpsellDial: {
 		//this test is used to dial down the upsell offer
-		datestamp: '20200421',
+		datestamp: '20200910',
```

Comment:
As far as I know changing the datestamp effectively creates a new / restarts the experiment. If this not intended it might be better to leave this as is.

Reply:
You're right, but also noting here that this is a fake A/B test. We are using an A/B test only to throttle the upsell offer, and there's no analysis coming out of it.

Reply:
Nice, that's good context to have! 🙂